File: Week1/김효준/요세푸스.py
N, K = map(int, input().split())
from collections import deque

arr = deque([i for i in range(1, N+1)])

print('<', end='')
while len(arr) > 1:
    for _ in range(K-1):
        arr.append(arr.popleft())
    print(arr.popleft(), end=", ")
print(*arr, end='>')

File: Week1/장수희/요세푸스.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf-8").trim().split("\n");

const [N,K] = input[0].split(" ").map(Number);

const arr = [];

for ( let i =0; i< N; i++){
    arr[i] = i+1;
}

let result = [];

while(arr.length){
    for ( let i =0; i< K - 1; i++){
       arr.push(arr.shift());
    }
    result.push(arr.shift())
}

console.log("<" + result.join(", ") + ">")